web/src/stream_topic_history.ts

```diff
@@ -216,7 +194,7 @@ export class PerStreamHistory {
             this.topics.set(topic_name, {
                 message_id,
                 pretty_name: topic_name,
-                historical: true,
+                count: 0,
```

Comment:
Why is 0 the correct value for `count` here? I think `1` might be more correct, under the theory that we know there's at least 1 message there, but nothing else about it.

I don't see how it matters given how we use it...

Reply:
Oh, it's because we add 1 via the next line, never mind :)